conda/testing/__init__.py

```diff
@@ -156,3 +165,94 @@ def conda_check_versions_aligned():
         )
         with open(version_file, "w") as fh:
             fh.write(version_from_git)
+
+
+@dataclass
+class CondaCLIFixture:
+    capsys: CaptureFixture
+
+    def __call__(self, *argv: str, no_capture: bool = False) -> tuple[str, str, int]:
+        """Mimic what is done in `conda.cli.main.main`"""
+        # drop the conda argument if provided
+        if argv[0] == "conda":
+            argv = argv[1:]
+
```

Comment:
needs test

Reply:
resolved by removing check, no value in being extra cautious here